trace/trace_test.go

```diff
@@ -510,6 +520,7 @@ func TestSetSpanStatus(t *testing.T) {
 			TraceID:      tid,
 			SpanID:       SpanID{},
 			TraceOptions: 0x1,
+			Tracestate:   &Tracestate{},
```

Comment:
The zero value of *Tracestate (nil) should be a valid empty Tracestate in the spirit of making the zero value useful. So it should not be necessary to set it here.